pkg/endpointmanager/conntrack.go

```diff
@@ -29,6 +29,16 @@ import (
 const (
 	// MinGcInterval is the minimum garbage collection interval.
 	MinGcInterval int = 5
+
+	// Global string for refering in metrics
+	Global string = "global"
```

Comment:
Is this still used?
Does this need to be exported? 
What type of label is this? Is this a scope? In that case it would be helpful if at least the comment described this, for example:

`Global determines the scope of conntrack metrics to be for an entire garbage collection round of the the global, shared conntrack map.`

Reply:
sorry, didn't push without that :-(